services/QuillLMS/spec/services/classlink_integration/instructor_linker_spec.rb

```diff
@@ -37,6 +37,17 @@
         expect(schools_admin.school_id).to eq(school.id)
       end
 
+      it 'updates the SchoolsAdmins record' do
+        old_school = create(:school)
+        create(:schools_admins, user: admin_user, school: old_school)
+        expect do
+          subject.run
```

Comment:
A style comment

`expect { subject.run }.not_to change(SchoolsAdmins, :count)`

There a bunch of these throughout the spec.  Three lines take up much more space than one and I don't think they provide more readability for simple expectation blocks.